project/cmake/modules/FindCpluff.cmake

```diff
@@ -22,7 +22,7 @@ if(NOT WIN32)
                                         --host=${ARCH}
                                         CFLAGS=${defines}
                                         LDFLAGS=${ldflags}
-                      BUILD_COMMAND make V=1)
+                      BUILD_COMMAND ${CMAKE_MAKE_PROGRAM} V=1)
```

Comment:
@Fneufneu: I just figured out that this breaks the build if one uses the ninja generator. [CMAKE_MAKE_PROGRAM](https://cmake.org/cmake/help/latest/variable/CMAKE_MAKE_PROGRAM.html) is set to a generator specific tool. But here and also for ffmpeg, we need to use "make". I'll investigate in a fix.